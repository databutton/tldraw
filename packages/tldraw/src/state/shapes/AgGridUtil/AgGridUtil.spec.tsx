import { AgGrid } from '..'

describe('AGGrid shape', () => {
  it('Creates a shape', () => {
    expect(AgGrid.create({ id: 'aggrid' })).toMatchSnapshot('aggrid')
  })
})
