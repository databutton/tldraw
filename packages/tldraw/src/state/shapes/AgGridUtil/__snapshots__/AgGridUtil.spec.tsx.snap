// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AGGrid shape Creates a shape: aggrid 1`] = `
Object {
  "assetId": "assetId",
  "childIndex": 1,
  "id": "aggrid",
  "name": "AgGrid",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "size": Array [
    1,
    1,
  ],
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": true,
    "scale": 1,
    "size": "small",
  },
  "type": "aggrid",
}
`;
